scripts/pre_commit_linter.py

```diff
@@ -102,12 +102,47 @@
         'regexp': r"\b(ddescribe|fdescribe)\(",
         'message': "In tests, please use 'describe' instead of 'ddescribe'"
                    "or 'fdescribe'",
-        'excluded_files': ()
+        'excluded_files': (),
+        'excluded_dirs': ()
     },
     {
         'regexp': r"\b(iit|fit)\(",
         'message': "In tests, please use 'it' instead of 'iit' or 'fit'",
-        'excluded_files': ()
+        'excluded_files': (),
+        'excluded_dirs': ()
+    },
+    {
+        'regexp': r"templateUrl: \'",
+        'message': "The directives must be directly referenced.",
+        'excluded_files': (
+            'core/templates/dev/head/pages/exploration_editor/'
+            'editor_tab/StateResponses.js',
+            'core/templates/dev/head/pages/exploration_player/'
+            'FeedbackPopupDirective.js'
+            ),
+        'excluded_dirs': (
+            'extensions/')
```

Comment:
You can move this to the previous line. Breaking after '(' is only needed if the contents of (...) span multiple lines.

Also, if you want this to be a 1-tuple, I think you need a comma after the string. That said, it might be simpler to just use lists instead.

Reply:
done.

oh, added the comma.